x-pack/filebeat/module/aws/s3_server_access_log/_meta/fields.yml

```diff
@@ -0,0 +1,97 @@
+- name: s3_server_access_log
+  type: group
+  description: >
+    Fields for AWS S3 server access logs.
+  fields:
+    - name: bucket_owner
+      type: keyword
+      description: >
+        The canonical user ID of the owner of the source bucket.
+    - name: bucket
+      type: keyword
+      description: >
+        The name of the bucket that the request was processed against.
+    - name: remote_ip
+      type: ip
+      description: >
+        The apparent internet address of the requester.
+    - name: requester
+      type: keyword
+      description: >
+        The canonical user ID of the requester, or a - for unauthenticated requests.
+    - name: request_id
+      type: keyword
+      description: >
+        A string generated by Amazon S3 to uniquely identify each request.
+    - name: operation
+      type: keyword
+      description: >
+        The operation listed here is declared as SOAP.operation, REST.HTTP_method.resource_type, WEBSITE.HTTP_method.resource_type, or BATCH.DELETE.OBJECT.
+    - name: key
+      type: keyword
+      description: >
+        The "key" part of the request, URL encoded, or "-" if the operation does not take a key parameter.
+    - name: request_uri
+      type: keyword
+      description: >
+        The Request-URI part of the HTTP request message.
+    - name: http_status
+      type: long
+      description: >
+        The numeric HTTP status code of the response.
+    - name: error_code
+      type: keyword
+      description: >
+        The Amazon S3 Error Code, or "-" if no error occurred.
+    - name: bytes_sent
+      type: long
+      description: >
+        The number of response bytes sent, excluding HTTP protocol overhead, or "-" if zero.
+    - name: object_size
+      type: long
+      description: >
+        The total size of the object in question.
+    - name: total_time
+      type: long
+      description: >
+        The number of milliseconds the request was in flight from the server's perspective.
+    - name: turn_around_time
+      type: long
+      description: >
+        The number of milliseconds that Amazon S3 spent processing your request.
+    - name: referrer
+      type: keyword
+      description: >
+        The value of the HTTP Referrer header, if present.
+    - name: user_agent
+      type: keyword
+      description: >
+        The value of the HTTP User-Agent header.
+    - name: version_id
+      type: keyword
+      description: >
+        The version ID in the request, or "-" if the operation does not take a versionId parameter.
+    - name: host_id
+      type: keyword
+      description: >
+        The x-amz-id-2 or Amazon S3 extended request ID.
```

Comment:
Is `host_id` a good name for the `x-amz-id-2` header?

Reply:
<img width="913" alt="Screen Shot 2019-08-08 at 7 15 39 AM" src="https://user-images.githubusercontent.com/14081635/62706250-5f8b7800-b9ac-11e9-94b0-15f999d960ad.png">
Yeah hmm it's from the aws documentation for access log format.